compliance-monitor/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1
+FROM python:3.10-alpine
```

Comment:
In my experience/remembrance, the use of Alpine Linux only saves a little space when creating container images, but carries the risk that its LIBC implementation does not positively influence the threading behavior of Python.
See also: https://pythonspeed.com/articles/alpine-docker-python/